Userland/Libraries/LibConfig/Client.cpp

```diff
@@ -19,6 +19,11 @@ Client& Client::the()
     return *s_the;
 }
 
+void Client::enable_permissive_mode()
+{
+    async_enable_permissive_mode();
```

Comment:
Wouldn't having this as an async method open it up for possible race conditions? I think this would be better suited as a sync method.